packages/core/lib/console.js

```diff
@@ -42,6 +42,7 @@ class Console extends EventEmitter {
       "build_directory"
     ]);
 
+    this.context = {};
```

Comment:
`this.context` and `this.repl.context` seem to have overlapping or the same concern. What do you think?

Reply:
You, know, I think you're right. I had originally left `this.context` because I thought we might need it elsewhere but I believe we don't. Change incoming!

Reply:
Ok, this concern should be addressed as of my last commit!